src/test/deps-index_test.ts

```diff
@@ -89,5 +90,26 @@ suite('Bundler', () => {
           serializeMap(index.entrypointToDeps),
           serializeMap(expectedEntrypointsToDeps));
     });
+
```

Comment:
This test doesn't prove anything was broken.  I wrote it to demonstrate this case was not broken, to help zero in on the root cause.